src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -174,6 +214,20 @@ function createModel(program: Program): any {
                 modelMap,
                 enumMap
             );
+            if (
+                contentTypeParameter &&
+                op.Parameters.some(
+                    (value) => value.Location === RequestLocation.Body
+                ) &&
+                !op.Parameters.some(
```

Comment:
So looks like we won't add the content type parameter if it's already defined in the operation parameters? It seems a defensive fix just in case service team forget to define content type parameter?

Reply:
global consumes/produces parameter will apply to all operations. if one operation defines its own content-type, it will override global setting.